store/query/utxo.go

```diff
@@ -67,7 +67,7 @@ func NewOutputQuerier(outputStore store.OutputStore) sdk.Querier {
 			return data, nil
 		case QueryTxOutput:
 			if len(path) != 2 {
-				return nil, sdk.ErrUnknownRequest("expected txo/<position>")
+				return nil, sdk.ErrUnknownRequest("expected utxo/<position>")
```

Comment:
What is being requested isn't necessarily unspent

Reply:
also it should be `output/<position>`

Reply:
ohh gotcha. I thought `txo` was a typo. I'll change this

Reply:
what is the difference between QueryTxInput/QueryTxOutput?

output query becomes very verbose. `/output/output/<pos>`